src/components/ServerVersion.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
+import { Icon, Tooltip } from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { fetchSettings } from "api/server";
+
+const Navigation: FC = () => {
+  const { data: settings } = useQuery({
+    queryKey: [queryKeys.settings],
+    queryFn: fetchSettings,
+  });
+
+  const serverVersion = settings?.environment?.server_version;
+  const majorVersion = serverVersion?.includes(".")
+    ? serverVersion.split(".")[0]
+    : undefined;
+  const isOldVersion = majorVersion ? parseInt(majorVersion) < 5 : false;
+  const versionLabel = serverVersion ? `Server ${serverVersion}` : undefined;
+
+  return serverVersion ? (
+    <>
+      <hr className="p-side-navigation__list is-dark navigation-hr" />
+      <li className="p-side-navigation__link server-version">
+        {isOldVersion ? (
```

Comment:
These nested `?` operators are making the code harder to read, can we replace them with the `&&` syntax, e.g.:
```suggestion
  return serverVersion && (
    <>
      <hr className="p-side-navigation__list is-dark navigation-hr" />
      <li className="p-side-navigation__link server-version">
        {isOldVersion && (
```

Reply:
I added an early exit above and restructured the rest a bit to make it easier to digest.